lib/Models/Catalog/Gtfs/GtfsCatalogItem.ts

```diff
@@ -475,3 +529,19 @@ function updateBbox(lat: number, lon: number, rectangle: RectangleExtent) {
   if (lon > rectangle.east) rectangle.east = lon;
   if (lat > rectangle.north) rectangle.north = lat;
 }
+
+/**
+ *
+ * @param data Object
+ * @param path Path in period-separated form. E.g. vehicle.position.compass
+ * @returns The value at the path inside data
+ */
+function jsonPathLike(data: JsonObject, path: string): any {
+  if (data === undefined) return undefined;
+  const index = path.indexOf(".");
+  if (index === -1) return data[path];
+  return jsonPathLike(
+    data[path.slice(0, index)] as JsonObject,
+    path.slice(index + 1)
+  );
+}
```

Comment:
There's also the [lodash.get](https://lodash.com/docs/4.17.15#get) function we could use.

Reply:
Ahh, I thought there must be an easy way to do this, but forgot to check lodash. Fixed.